enhancements/single-node-production-edge-cluster-profile.md

```diff
@@ -0,0 +1,469 @@
+---
+title: single-node-production-edge-cluster-profile
+authors:
+  - "@dhellmann"
+  - "@eranco"
+  - "@romfreiman"
+  - "@markmc"
+reviewers:
+  - TBD, probably all leads
```

Comment:
can we get this updated to make sure that affected operators/groups get to review this